src/cpp/desktop/DesktopGwtCallback.cpp

```diff
@@ -1274,8 +1274,7 @@ double GwtCallback::getZoomLevel()
 
 void GwtCallback::setZoomLevel(double zoomLevel)
 {
-   options().setZoomLevel(zoomLevel);
-   desktopInfo().setZoomLevel(zoomLevel);
```

Comment:
Note that this was redundant (setting the preference in options also updates the desktopInfo instance) so it's been removed.